### ISSUE TITLE
[v3.2.0-rc.5] Tab Body-Stil, Icon-Stil unterstützt kein Binding

### ISSUE THREAD

**Issue by mcuiobroker** [Zefau/ioBroker.jarvis#2819]
evtl Feature?
![image](https://github.com/user-attachments/assets/20595c14-3860-4992-8b90-fe215f5341b4)
Es wird kein Binding unterstützt?
Dachte es wäre schon auch für TAB und Seite.

**Comment by Zefau** [comment]
Läuft eigentlich durch die gleichen Routinen wie alle anderen Bindings. Hast du ein Beispiel? Dann probiere ich mal bei mir.

**Comment by mcuiobroker** [comment]
```
{"{0_userdata.0.jarvis.test.boolean4}=true":{"color":"orange"}}
```
![image](https://github.com/user-attachments/assets/8bdcb968-8e05-4ffb-840a-08094002af47)

![image](https://github.com/user-attachments/assets/30c309bd-7fa1-4245-88f5-c827b31c03a7)

sollte so aussehen
![image](https://github.com/user-attachments/assets/8bcfe11d-de00-4770-bbf7-eaa1dbd05269)

**Comment by mpl1337** [comment]
Hallo,

RC.12 ähnliches Problem:

Datenpunkt Stil:


Funktioniert:
`{
  "{0_userdata.0.test.testdp}=true": {
    "color": "#FF0000",
    "font-Weight": "bold"
  },
  "{0_userdata.0.test.testdp}=false": {
    "color": "#00ff00"
  }
}
`

Funktioniert nicht:
`{
  "{alias.0.Wohnzimmer.Luftqualität.Wohnzimmer_CO2_Schwellwert}=true": {
    "color": "#FF0000",
    "font-Weight": "bold"
  },
  "{alias.0.Wohnzimmer.Luftqualität.Wohnzimmer_CO2_Schwellwert}=false": {
    "color": "#00ff00"
  }
}
`

Funktioniert auch nicht:
`{
  "{openknx.0.lüftung_klima.Sensoren.Wohnzimmer_CO2_Schwellwert}=true&&{val}>0": {
    "color": "#FF0000",
    "font-Weight": "bold"
  },
  "{openknx.0.lüftung_klima.Sensoren.Wohnzimmer_CO2_Schwellwert}=false": {
    "color": "#00ff00"
  }
}
`


Sobald der Pfad nicht "0_userdata.0" ist, funktioniert es nicht.

**Comment by mpl1337** [comment]
Bin weiter gekommen.

Habe 

alias.0.Wohnzimmer.Luftqualit**ä**t.Wohnzimmer_CO2_Schwellwert
zu 
alias.0.Wohnzimmer.Luftqualit**ae**t.Wohnzimmer_CO2_Schwellwert 

geändert und den Alias entsprechend auch. von Ä zu AE abgeändert.

Die Routine kommt mit Umlauten nicht zurecht?

**Comment by intruder7777** [comment]
hier keine Umlaute und funktioniert auch nicht.
`  "({alias.0.Zigbee.Pflanzensensoren.Pflanzensensor_02.MOISTURE}<={alias.0.Zigbee.Pflanzensensoren.Pflanzensensor_02.MIN_MOISTURE}) || ({alias.0.Zigbee.Pflanzensensoren.Pflanzensensor_03.MOISTURE}<={alias.0.Zigbee.Pflanzensensoren.Pflanzensensor_03.MIN_MOISTURE})": {
    "color": "red"
  },
  "default": {
    "color": "green"
  }
}`

**Comment by uwe72** [comment]
> Bin weiter gekommen.
> 
> Habe
> 
> alias.0.Wohnzimmer.Luftqualit**ä**t.Wohnzimmer_CO2_Schwellwert zu alias.0.Wohnzimmer.Luftqualit**ae**t.Wohnzimmer_CO2_Schwellwert
> 
> geändert und den Alias entsprechend auch. von Ä zu AE abgeändert.
> 
> Die Routine kommt mit Umlauten nicht zurecht?

Hat es dann funktioniert? Gibt es einen funktionierenden Workaround? Habe keine Umlaute

**Comment by mpl1337** [comment]
Ja. Funktioniert seit dem. Workaround? Umlaute und Sonderzeichen meiden : D. Leerzeichen habe ich auch noch durch unterstriche ersetzt.

**Comment by uwe72** [comment]
> Ja. Funktioniert seit dem. Workaround? Umlaute und Sonderzeichen meiden : D. Leerzeichen habe ich auch noch durch unterstriche ersetzt.

ok, habe keine Umlaute/Unterstriche/Leerzeichen. Funktioniert bei mir leider nicht.

`0_userdata.0.sommerbetrieb`

![Image](https://github.com/user-attachments/assets/f188c82b-7aac-4d14-950b-35317e7b28f2)

`{"{0_userdata.0.sommerbetrieb}=false":{"background-color":"red"}}`

Mit Version (Instance: jarvis.0, v3.2.0-rc.12)

**Comment by tschortschde** [comment]
Dieses Problem tritt auch bei mir auf.
Mich würde wirklich sehr interessieren, wie es denn mit dem Adapter jetzt weitergeht.
Bitte um Rückantwort @Zefau
